sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -54,7 +46,7 @@
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-sdk</artifactId>
-      <version>0.2.0-SNAPSHOT</version>
+      <version>0.2.0</version>
```

Comment:
Does this need to list the dependency version as it should be determined transitively from the parent POM.

Reply:
I tried it without providing one but doesn't pick it up from the parent pom. Would it be because it is a test dependency?